### DISCUSSION TITLE
problem

### DISCUSSION THREAD

**Discussion by HaroMeng** [nvim-treesitter/nvim-treesitter#6918] [General]
Error detected while processing /home/haromeng/.config/nvim/init.lua:
Installation not possible: ...er/start/nvim-treesitter/lua/nvim-treesitter/install.lua:87
: Parser not available for language "help"
See https://github.com/nvim-treesitter/nvim-treesitter/#adding-parsers on how to add a ne
w parser!
Press ENTER or type command to continue

config:

├── init.lua
├── lua
│   ├── core
│   │   ├── keymaps.lua
│   │   └── options.lua
│   └── plugins
│       ├── autopairs.lua
│       ├── bufferline.lua
│       ├── cmp.lua
│       ├── comment.lua
│       ├── gitsigns.lua
│       ├── lsp.lua
│       ├── lualine.lua
│       ├── nvim-tree.lua
│       ├── plugins-setup.lua
│       ├── telescope.lua
│       └── treesitter.lua
└── plugin
    └── packer_compiled.lua
 init.lua
require("plugins.lualine")
require("plugins/nvim-tree")
require("plugins/treesitter")
require("plugins/lsp")
require("plugins/cmp")
require("plugins/comment")
require("plugins/autopairs")
require("plugins/bufferline")
require("plugins/gitsigns")
require("plugins/telescope")
treesitter.lua
require'nvim-treesitter.configs'.setup {

  ensure_installed = { "vim", "help", "bash", "c", "cpp", "javascript", "json", "lua", "python", "typescript", "tsx", "css", "rust", "markdown", "markdown_inline" }, -- one of "all" or a list of languages

  highlight = { enable = true },
  indent = { enable = true },

  rainbow = {
    enable = true,
    extended_mode = true,
    max_file_lines = nil,
  }
}

**Comment by clason** [comment]
The parser is named `vimdoc`, not `help`.

**Comment by HaroMeng** [comment]
thanks i have sloven the problem

**Reply by liliaoyuan** [reply]
Hi there! Can you share your sol? new to nvim and I came cross same problem ..

**Reply by HaroMeng** [reply]
delete the help

**Reply by clason** [reply]
The language name is `vimdoc`, not `help`.

**Reply by liliaoyuan** [reply]
Thanks, it works.
I guess it could be some out-date problem.
